suave/backends/redis_store_backend.go

```diff
@@ -29,6 +30,11 @@ type RedisStoreBackend struct {
 	cancel   context.CancelFunc
 	redisUri string
 	client   *redis.Client
+	local    *miniredis.Miniredis
+}
+
+func NewLocalConfidentialStore() *RedisStoreBackend {
```

Comment:
We can use the `miniredis` version of the `RedisStoreBackend` that works in memory as the local store.